engine-src/Game/LambdaHack/Client/UI/DrawM.hs

```diff
@@ -331,14 +331,14 @@ drawFrameActor drawnLevelId = do
                 in if hpCheckWarning || calmCheckWarning
                    then Color.Red
                    else bcolor
-         in Color.attrCharToW32 $ Color.AttrChar Color.Attr{..} symbol
+         in Color.attrCharToW32 $ Color.AttrChar Color.Attr{..} (DefsInternal.displayContentSymbol symbol)
       {-# INLINE viewProj #-}
       viewProj as = case as of
         aid : _ ->
           let ActorUI{bsymbol, bcolor} = sactorUI EM.! aid
               bg = Color.HighlightNone
               fg = bcolor
-         in Color.attrCharToW32 $ Color.AttrChar Color.Attr{..} bsymbol
+         in Color.attrCharToW32 $ Color.AttrChar Color.Attr{..} (DefsInternal.displayContentSymbol bsymbol)
```

Comment:
This one can be gotten from `Defs`, so it's better not to refer to the internal one.